### DISCUSSION TITLE
Potentiometers with death zones for Center Detent

### DISCUSSION THREAD

**Discussion by DEEWEESTUDIO** [shanteacontrols/OpenDeck#303] [Q&A]
Hi All, 

As you know midi goes from 0 to 127 and the center of a knob would be 64. I want to make a center detent button as found in dj mixers. But this for controlling the pitch of a synth. Let's say the middle is 0 on the oscillator turning it left is -12, turning it right goes + 12. 

I made a system to have a center detent without needing a center detent potentiometer, because the pots I want to use don't have a built-in center detent option. 

<img width="516" alt="Screenshot 2024-07-12 at 11 36 06" src="https://github.com/user-attachments/assets/ccf5ff32-93c2-419e-a5b9-196260247fa4">

But on these potentiometers the center isn't really "the center" value. It goes from 60 to 70 almost, there's always a margin...

Is it possible to e.g.  program/code: _for midi cc 12, if value = 60 it's 64, if value = 61 it's 64_ and so on. to make a death zone for the middle so during a show it doesn't always have to be exactly the center. 

Happy to receive any info/help on this. 

Kind Regards, 
Pim

**Comment by paradajz** [comment]
Not really at the moment, for this kind of thing you'd be better suited with something like MIDI translator:
https://www.bome.com/products/miditranslator
